src/transformers/models/speecht5/configuration_speecht5.py

```diff
@@ -359,8 +359,6 @@ class SpeechT5HifiGanConfig(PretrainedConfig):
     Args:
         model_in_dim (`int`, *optional*, defaults to 80):
             The number of frequency bins in the input log-mel spectrogram.
-        sampling_rate (`int`, *optional*, defaults to 16000):
-            The sampling rate at which the output audio will be generated, expressed in hertz (Hz).
```

Comment:
Not sure why this was deleted?

Reply:
Noticed it wasn't used, but just put it back in the interest of keeping this PR to just touching one model